SU-SWS/decanter-web
# ADAPT-3153 Update Identity Colors in Decanter Documentation

# READY FOR REVIEW

# Summary
- Update Identity Colors in Decanter Documentation

# Needed By (Date)
- tbd

# Urgency
- low

# Steps to Test
1. Review the Sass documentation:

Identity Color Docs:
- [Preview](https://deploy-preview-89--elegant-poitras-87214a.netlify.app/docs/scss/index.html#variable-variable-su-color-digital-red)
- [Live](https://decanter.stanford.edu/docs/scss/index.html#variable-variable-$su-color-cardinal-red)

Check to see that the preview pulls in the new Identity colors, i.e.
- $su-color-digital-red
- $su-color-digital-blue
- $su-color-foggy
- etc

2. Review the Color page documentation (this has been updated directly in the Netlify CMS):
- [Live](https://decanter.stanford.edu/page/brand-design-elements-color/)

# Associated Issues and/or People
- [ADAPT-3153](https://stanfordits.atlassian.net/browse/ADAPT-3153)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
